python/scripts/rfi_receiver/rfi_receiver.py

```diff
@@ -332,7 +327,10 @@ def data_listener(thread_id):
     mode = app.mode
     firstPacket = True
     vdifPacketSize = global_freq * 4 + RFIHeaderSize
-    chimePacketSize = RFIHeaderSize + 4 * local_freq
+    # CHIME packet: RFIHeader + frequency_bins[local_freq] + rfi_avg[local_freq]
+    chimePacketSize = RFIHeaderSize + 4 * local_freq + 4 * local_freq
```

Comment:
Currently the size and shape of the packet both depend on both sizes having the same config.  Could we add some checks on this side to make sure the contents of the header match what this config is set to (e.g. for local_freq, etc?

Reply:
So there already exists a `HeaderCheck` function on line 253.

Reply:
Is that what you mean?

Reply:
Ah perfect!